### PR TITLE
added navigation component

### PR DESCRIPTION
**Pull request by anshikaa12**
>it is responsive with hamburger menu

### CODE REVIEW

File: documentation/documentation.html [hunk 1]
```diff
@@ -1482,23 +1482,25 @@ <h2 class="main-h2">Example of Stacked List</h2>
...
-                                <h4 class="card-title">iPhone 13</h4>
-                                <span class="card-subtext">Why iPhone?
-                                    <br>
-                                    There's no phone like iPhone.
-                                </span>
+                            <div class="basic-card">
+                                <div class="card-header">
+                                    <img class="card-img-header" src="../components/cards/images/iphoneimg.jpg"
+                                        class="card-img">
+                                    <div class="header-text">
```
Comment (ashishnnnnn) [inline documentation/documentation.html:1489]: You can add alt attribute to the images

**Review by ashishnnnnn — APPROVED** [review]
You can try that suggestion.. the rest of code looks fine ..

**Review by kedark152 — APPROVED** [review]
Looks Good.

File: documentation/documentation.html (hunk 1 above)
Comment (anshikaa12) [reply documentation/documentation.html:1489]: sure will do  that

**Pull request merged**